lib/backbone-state-tracking.js

```diff
@@ -4,22 +4,27 @@
 var LOADING_EVENTS = 'request';
 var LOADED_EVENTS = 'sync error reset';
 
-function loadingChange(obj, newState) {
+function loadingChange(obj, newState, options) {
   newState = !!newState;
   var current = !!obj.loading;
+
+  if(options && options.method !== 'read') {
```

Comment:
This logic doesn't feel very obvious to me.

For one, where does this options hash get set? Is it in backbone? Or sync mixin? It would be worth documenting this.

Secondly, this doesn't seem like it would handle something like the following sequence of events:

* `request`: `method=read` (sets `loading=true`)
* `request` `method=create` (sets `loading=false`)
* At this point, the read is still taking place, but the create has cancelled the loading state, so loading is wrong.
* `sync`: `method=create` (again, sets `loading=false`)
* `sync`: `method=read` (again, sets `loading=false`) 


What you should be doing, I think, is ignoring all events except those for reads and not changing the loading state for them (instead of setting the state to false when a `create` occurs).

Even better would be to count up by one for each `read request` and count down for `read sync|error` and set `loading=true` when the count is greater than zero and `loading=false` when count is zero. `reset` events would always set count to zero and reset the `loading` flag to `false`.